components/identifier-generator/front/src/feature/pages/ListPage.tsx

```diff
@@ -69,6 +71,14 @@ const ListPage: React.FC<ListPageProps> = ({onCreate}) => {
     return identifierAttributes.find(attribute => attribute.code === target)?.label;
   };
 
+  const filterOnLabelOrCode =
+    (searchValue: string, locale: LocaleCode) =>
+    (entity: {code: string; labels: LabelCollection}): boolean =>
+      -1 !== entity.code.toLowerCase().indexOf(searchValue.toLowerCase()) ||
+      (undefined !== entity.labels[locale] &&
+        -1 !== entity.labels[locale].toLowerCase().indexOf(searchValue.toLowerCase()));
+  const filteredGenerators = generators.filter(filterOnLabelOrCode(search, locale));
```

Comment:
Just a suggestion, tell me if it looks clearer or not like this:
```suggestion
  const filteredGenerators = useMemo(() => generators.filter(({code, labels}) => 
    code.toLowerCase().includes(searchValue.toLowerCase()) || labels[locale]?.toLowerCase()?.includes(searchValue.toLowerCase())
  ), [generators, locale, searchValue];
```